ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.collection.less

```diff
@@ -0,0 +1,6 @@
+@{customElementNamespace}result-form-collection {
+  > .collection {
```

Comment:
We'll need to have a display for `result-form-collection` set or we'll get some weird issues on certain browsers.  I would suggest `display: block`